src/server_state.cpp

```diff
@@ -1364,6 +1364,10 @@ void wsrep::server_state::state(
 
     if (allowed[state_][state] == false)
     {
+        /* If we are disconnecting ignore any remaining state events from
```

Comment:
For readability, would it be better to have this as a guard clause at the very beginning of the method? Now the reader must check from the allowed transitions matrix if there are possibly allowed transitions which might bypass this check.